src/meta/meta_bulk_load_service.cpp

```diff
@@ -293,6 +293,7 @@ void bulk_load_service::create_partition_bulk_load_dir(const std::string &app_na
 {
     partition_bulk_load_info pinfo;
     pinfo.status = bulk_load_status::BLS_DOWNLOADING;
+    pinfo.ever_ingest_succeed = false;
```

Comment:
has the last status stored in zk? if stored, can you use the status, but no define new variable?

Reply:
`last status` won't store in zk, only current status will be stored.

Reply:
if current status stored, when failed and re-start,  will it change to `last_status`?